services/dynamic-sidecar/Dockerfile

```diff
@@ -67,16 +72,22 @@ FROM base as build
 
 ENV SC_BUILD_TARGET=build
 
-RUN apt-get update &&\
-  apt-get install -y --no-install-recommends \
-  build-essential
+RUN --mount=type=cache,id=basecache,target=/var/cache/apt,mode=0755,sharing=locked \
```

Comment:
Q: what is the purpose of this mount?

Reply:
caching from the host to the docker build. this improves build speed.